src/content/ja/2020/capabilities.md

```diff
@@ -0,0 +1,332 @@
+
+`navigator.wakeLock.request()`メソッドはウェイクロックを作成する。このメソッドは `WakeLockType`パラメーターを取ります。将来的には、Wake Lock APIは画面をオフにしてCPUをオンにしたままにするなど、他のロックタイプを提供できるようになるかもしれません。今のところ、APIはスクリーンロックのみをサポートしているので、デフォルト値が `'screen'` のオプション引数を1つだけ用意しています。このメソッドは`WakeLockSentinel`オブジェクトに解決するプロミスを返す。開発者はこの参照を保存して`release()`メソッドを呼び出し、後で画面のウェイクロックを解除する必要があります。ブラウザはタブが非アクティブになったり、ユーザがウィンドウを最小化したりすると自動的にロックを解除します。またブラウザは、例えばバッテリー残量が少ないなどの理由で、要求を拒否して約束を拒否することがあります。
+
+{{ figure_markup(
+  image="screen_wake_lock_api.png",
+  caption="Screen Wake Lock APIを利用しているページ数",
+  description="HTTP Archiveで監視しているページ数に基づいて、デスクトップページとモバイルページを比較したScreen Wake Lock APIの利用状況のグラフです。2020年10月時点では、デスクトップ10ページ、モバイル5ページでAPIが利用されています。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxqot9ALgxcgOVJntkzIKnkpo3idIPy-tL0t_nzC5BwFuq0ThgK5OXOYVVOpama4vB2EyggX813d33/pubchart?oid=718278185&format=interactive",
+  sheets_gid="1008442251",
+  sql_file="wake_lock_acquire_screen_lock_usage.sql"
+  )
+}}
+
+米国で人気の料理サイトBettyCrocker.comは、Screen Wake Lock APIの助けを借りて、料理中に画面が暗くなるのを防ぐオプションをユーザーに提供している。<a lang="en" href="https://web.dev/betty-crocker/">ケーススタディ</a>では平均セッション時間が通常より3.1倍長くなり、バウンス率が50%減少し、購入意向指標が約300%増加したと発表しています。このように、インターフェイスは、Webサイトやアプリケーションの成功に直接的に測定可能な効果を持っています。Screen Wake Lock APIは、2020年7月にGoogle Chrome 84で提供されました。HTTP Archiveには4月、5月、8月、9月、10月のデータしかありません。Chrome 84のリリース後、利用率は一気に上昇しました。2020年10月には、デスクトップ10ページ、モバイル5ページでAPIが採用されました。
+
+## アイドル検出API
+
+アプリケーションの中には、ユーザーがデバイスを積極的に使用しているか、アイドル状態にあるかを判断する必要があります。例えば、チャットアプリケーションは、ユーザーが不在であることを表示することがあります。画面やマウス、キーボードとのやりとりがないなど、様々な要因が考慮されます。<a lang="en" href="https://web.dev/idle-detection/">アイドル検出API</a>(<a lang="en" href="https://wicg.github.io/idle-detection/">WICGコミュニティグループ報告書</a>)では、ある閾値を設定することで、ユーザがアイドル状態か画面ロック状態かを確認することができる抽象的なAPIを提供しています。
+
+これを実現するために、APIはグローバルな`window`オブジェクト上に新しい`IdleDetector`インターフェイスを提供します。開発者がこの機能を利用する前に、まず`IdleDetector.requestPermission()`を呼び出してパーミッションを要求しなければなりません。ユーザが許可を与えれば、開発者は`IdleDetector`の新しいインスタンスを作成できます。このオブジェクトは2つのプロパティを提供します。`userState`と`screenState`の2つのプロパティを提供します。このオブジェクトは、ユーザの状態と画面の状態のどちらかが変化したときに`change`イベントを発生させます。最後に、アイドル検出は`start()`メソッドを呼び出して起動する必要があります。このメソッドは2つのパラメーターを持つ設定オブジェクトを受け取ります。ユーザーがアイドル状態でなければならない時間をミリ秒単位で定義する`threshold`と、開発者はオプションで`AbortSignal`を`abort`プロパティに渡すことができます。
+
+{{ figure_markup(
+  image="idle_detection_api.png",
+  alt="アイドル検出APIを使用したChromeでのページロードの割合",
+<<<<<<< HEAD
+  caption='アイドル検出APIを使用したChromeでのページロードの割合<br>(出典. <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">アイドル検出</a>。)',
+=======
+  caption='アイドル検出APIを使用したChromeでのページロードの割合<br>(出典. <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">アイドル検出</a>.)',
+>>>>>>> 7cbbddd1d53bdd68c54abbf24c7283a5c6bf896c
```

Comment:
Looks like you have a merge conflict here @ksakae1216 so it has both lines and is waiting for you to choose which one to us:

```suggestion
  caption='アイドル検出APIを使用したChromeでのページロードの割合<br>(出典. <a href="https://chromestatus.com/metrics/feature/timeline/popularity/3017">アイドル検出</a>。)',
```

BTW if you click on the error you'll be given a clue as to why it's failing:

![image](https://user-images.githubusercontent.com/10931297/100727195-d2004280-33bd-11eb-9637-95b9f457f6d7.png)

Line numbers won't match up completely as this is the generated HTML but hopefully enough context to give you a clue. In this case I searched for "idle_detection_api.png" and saw the error.

Reply:
@bazzadp Thanks, I see!!
Next time, I carefully confirm log.